libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/TreeViewRenderer/TreeViewNode.tsx

```diff
@@ -33,13 +33,31 @@ export class TreeViewNode extends React.Component<ITreeViewNodeProps> {
     const classNames = treeViewRendererStyles();
     const gradientFillId = getRandomId();
     const theme = getTheme();
+    const nodeFilterProps = node.data.filterProps;
+    const ariaLabel = `${nodeFilterProps.numCorrect} ${
+      nodeFilterProps.numIncorrect
+    } ${nodeFilterProps.errorCoverage.toFixed(
+      2
+    )} ${nodeFilterProps.metricValue.toFixed(2)}`; // TODO: waiting for PM input
+    let strokeWidth = 1.5;
+    if (this.props.node.data.nodeState.onSelectedPath) {
+      if (node.data.nodeState.isSelectedLeaf) {
+        strokeWidth = 6;
+      } else {
+        strokeWidth = 3;
+      }
+    }
     return (
       <>
         <g
+          onFocus={this.onSelect}
           style={node.data.nodeState.style}
           onClick={this.onSelect}
           pointerEvents="all"
           ref={this.ref}
+          tabIndex={0}
```

Comment:
It seems this is currently tabbing through the nodes, but I think we want to just go to the current selected node and then, if the user wants to change nodes, they can use up/down arrows - however we shouldn't be going to every single node in the tree when pressing tab key, which is what I am seeing in the dashboard app right now.

Reply:
This can also be fixed in a separate PR though, doesn't need to be addressed right away.

Reply:
maybe one of these might help:
https://codereview.stackexchange.com/questions/243627/handling-arrow-keys-and-tabindex
https://stackoverflow.com/questions/39733783/tab-to-next-tabindex-field-on-down-arrow-key

Reply:
Thanks Ilya, Will look into arrow navigation after we finished everything(accessibility) required by GA